libs/block-editor/src/lib/demo/block-editor.component.html

```diff
@@ -1,4 +1,5 @@
 <tiptap-bubble-menu [editor]="editor">
```

Comment:
Little bug:

https://user-images.githubusercontent.com/751424/142462537-65f1930e-c43b-4d4d-a6b5-7d5b3a5b0907.mp4

Reply:
Tippy showing twice.

https://user-images.githubusercontent.com/751424/142465220-1dd06b15-5c24-4277-846c-58278c50562d.mp4

Reply:
Mmm... I'll take a look at that.

Reply:
Also, we need to make that when the selected text is an URL we pre-populate the link input with that url.